src/iop/lut3d.c

```diff
@@ -790,6 +790,18 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   }
 }
 
+void filepath_set_OS_separator(char *filepath)
```

Comment:
I would just replace all \ to / on Windows as / is also supported on Windows. This will does nothing on Linux as there is no \ in PATH. It will be simpler and won't need OS dependent code.

Reply:
Excellent ! Done